webrender_api/src/api.rs

```diff
@@ -358,9 +430,15 @@ pub struct AddFontInstance {
     pub variations: Vec<FontVariation>,
 }
 
+// TODO: Split this in two emums.
+// I am postponing this because while it is a trivial change, it is a hard one to rebase.
```

Comment:
Will you do this now? If not, probably better to remove this comment and open a bug tracking this.